src/scrabbleScore.js

```diff
@@ -24,17 +23,22 @@ let SCRABBLE_TILE_SCORES = {
  */
 
 function scrabbleScore(word) {
-  // This is your job. :)
+  let score = 0;
+  let wordArray = word.toLowerCase().split('');
+
+  for(let letter of wordArray){
```

Comment:
You can use `for ... of` on a string directly and you'll get one letter at a time, so:

```javascript
for (let char of word.toLowerCase()) {
  // ...
}
```